applications/plugins/spi_mem_manager/lib/spi/spi_mem_chip.c

```diff
@@ -0,0 +1,81 @@
+#include "spi_mem_chip.h"
+
+const SPIMemChipVendorName SPIMemChipVendorNames[] = {
```

Comment:
We use snake case for variables and function names, so `spi_mem_chip_vendor_names`.